source/administration/server-side-encryption/server-side-encryption-sse-kms.rst

```diff
@@ -157,23 +157,22 @@ MinIO server host in the deployment:
    * - :envvar:`MINIO_KMS_KES_ENDPOINT`
      - The endpoint for the MinIO ``Play`` KES service.
 
-   * - :envvar:`MINIO_KMS_KES_KEY_FILE`
-     - The private key file corresponding to an :kes-docs:`identity <concepts/#authorization>` on the KES service. 
-       The identity must grant permission to create, generate, and decrypt keys. 
-       Specify the same identity key file as the ``KES_KEY_FILE`` environment variable in the previous step.
+   * - :envvar:`MINIO_KMS_KES_API_KEY`
+     - The API key :kes-docs:`generated by KES <tutorials/kes-for-minio/#kes-server-setup>` for the MinIO deployment.
+       The identity of the API key must grant permission to create, generate, and decrypt keys. 
 
-   * - :envvar:`MINIO_KMS_KES_CERT_FILE`
-     - The public certificate file corresponding to an :kes-docs:`identity <concepts/#authorization>` on the KES service. 
-       The identity must grant permission to create, generate, and decrypt keys. 
-       Specify the same identity certificate as the ``KES_CERT_FILE`` environment variable in the previous step.
+       The API key is the preferred way to authenticate with the KES server.
+       If circumstances require it, specify the :envvar:`MINIO_KMS_KES_KEY_FILE` and :envvar:`MINIO_KMS_KES_CERT_FILE` instead.
```

Comment:
Hm - this begs the question of 'wait what circumstances'

Maybe:

- KES API Keys allow for a streamlined and secure authentication process to the KES server.

- You can still use the TLS Certificate-based authentication scheme via X and Y

- API keys are mutually exclusive with certificate-based authentication.

- This procedure uses API keys..